managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/DimensionStats.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.bookkeeper.mledger.util;
+
+import org.HdrHistogram.Histogram;
+import org.HdrHistogram.Recorder;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ */
+public class DimensionStats {
+
+    /** Statistics for given dimension **/
+    public double meanDimensionMs;
+
+    public double medianDimensionMs;
+
+    public double dimension95Ms;
+
+    public double dimension99Ms;
+
+    public double dimension999Ms;
+
+    public double dimension9999Ms;
+
+    public double dimensionCounts;
+
+    public double elapsedIntervalMs;
+
+    private Recorder dimensionTimeRecorder = new Recorder(TimeUnit.MINUTES.toMillis(10), 2);
```

Comment:
Using a very high max value, reduces a lot the precision of the computed quantiles. I would suggest to use max 30sec and cap the latency to 30sec before sending to the recorder.